install/0001_00_cluster-version-operator_03_service.yaml

```diff
@@ -1,16 +1,19 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: cluster-version-operator
+  name: metrics
   namespace: openshift-cluster-version
   labels:
     k8s-app: cluster-version-operator
   annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: openshift-cluster-version-serving-cert
     exclude.release.openshift.io/internal-openshift-hosted: "true"
 spec:
   type: ClusterIP
   selector:
     k8s-app: cluster-version-operator
   ports:
-  - name: metrics
-    port: 9099 # chosen to be in the internal open range
+  - name: https
+    port: 443
```

Comment:
To back this, doesn't the CVO need to be serving HTTPS on 443?  How does it get the X.509 keypair and cert to do that?  Also, you'll probably need to adjust something in [here][1] to set [`--listen`][2] or change [the default][3].

[1]: https://github.com/openshift/cluster-version-operator/blob/fa452c2d270f1f989f3868ef97ae8cf825713583/install/0000_00_cluster-version-operator_03_deployment.yaml#L25-L29
[2]: https://github.com/openshift/cluster-version-operator/blob/fa452c2d270f1f989f3868ef97ae8cf825713583/install/0000_00_cluster-version-operator_03_deployment.yaml#L25-L29
[3]: https://github.com/openshift/cluster-version-operator/blob/ac394f1f73a6e1367d0af8622d4f9af545a6e39a/pkg/start/start.go#L86

Reply:
> To back this, doesn't the CVO need to be serving HTTPS on 443?

*This* portion of my comment is really about `targetPort`, and now there's an [appropriately positioned thread][1] for that.

About the external-facing `port`, we do have [existing docs][2] around host-port 9099 being used for the CVO, so we might want to leave that alone instead of moving it to 443.

[1]: https://github.com/openshift/cluster-version-operator/pull/358#discussion_r417057773
[2]: https://github.com/openshift/openshift-docs/blame/d646e0822ca61e1e5389ea80441d1389b5fd13bf/modules/installation-network-user-infra.adoc#L61

Reply:
Fixed as of 16da75689